### PR TITLE
Dynamic Commands 3

### PR DESCRIPTION
**Pull request by AlphaModder**
Continuation of #539 for zml's new API. I'll write a description in more detail when it isn't 2 AM...

### CODE REVIEW

File: src/main/java/org/spongepowered/api/service/command/AliasContext.java [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package org.spongepowered.api.service.command;
+
+import com.google.common.base.Optional;
+import org.spongepowered.api.util.command.CommandSource;
+
+
```
Comment (simon816) [inline src/main/java/org/spongepowered/api/service/command/AliasContext.java]: Missing javadocs

File: src/main/java/org/spongepowered/api/util/command/dispatcher/TranslationPlaceholder.java [hunk 2]
```diff
@@ -31,7 +31,7 @@
 /**
  * This class contains methods intended for use as placeholders until true localization is implemented for SpongeAPI strings.
  */
-class TranslationPlaceholder {
+public class TranslationPlaceholder {
```
Comment (zml2008) [inline src/main/java/org/spongepowered/api/util/command/dispatcher/TranslationPlaceholder.java]: please don't -- just copy the class if you need it in a different package. This is a temporary thing, needs to be replaced by a real translation system eventually.

Comment (AlphaModder) [inline src/main/java/org/spongepowered/api/util/command/dispatcher/TranslationPlaceholder.java]: @zml2008 ok.

**Comment by AlphaModder** [comment]
No reason for this anymore.

**Pull request closed without merging**